src/components/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,278 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import Typography from "./Typography";
+
+export default {
+  title: "Base/Typography",
+  component: Typography,
+  parameters: {
+    componentSubtitle: "Typography component has all text variants available",
+    docs: {
+      description: {
+        component:
+          "<p>To use this compoent from our Storyboook, import it from <code>@mui-agronod</code> and not from materialUi, since we added our base theme overrides for different headlines and other fonts</p><p>Use typography to present your design and content as clearly and efficiently as possible.</p><p>For typography we use object defined in <code>baseTheme</code>.</p><a target='_blank' href='https://material.io/design/typography/the-type-system.html'>Go to material design for more</a>",
```

Comment:
I think you are right, in this case we don't need to have component in storybook here since it gets style overrides already and we should import it from @mui/material.